test/gurka/test_probable_restrictions.cc

```diff
@@ -67,7 +67,8 @@ class ProbableRestrictions : public testing::TestWithParam<int> {
               {gurka::node_member, "I", "via"},
           },
           {{"type", "restriction"}, {"restriction:probable", "no_left_turn @ probability=0"}}}};
-    map = gurka::buildtiles(layout, ways, {}, relations, "test/data/gurka_time_probable_restrictions",
+    map = gurka::buildtiles(layout, ways, {}, relations,
+                            VALHALLA_BUILD_DIR "test/data/gurka_time_probable_restrictions",
```

Comment:
yeah I'm confused.. isn't this the graph directory? don't we set that now in make_config()?

Reply:
same in lots of other places in gurka

Reply:
we do and I went through 2 rounds of this and missed this in the last round. it's surprisingly annoying to find all occurances at midnight after a long day at work hahaha